includes/utilities.inc

```diff
@@ -106,7 +106,7 @@ function islandora_entities_create_citation_tab(AbstractObject $object) {
     $simplexml = simplexml_load_string($mads);
     $identifiers = $simplexml->identifier;
     foreach ($identifiers as $identifier) {
-      if ($identifier['type'] == 'u1') {
+      if ($identifier['type'] == 'u1' && (string) $identifier) {
```

Comment:
Same deal here as below on islandora_entities.module:615; I dunno why I did this from the bottom up. Something to do with being left-handed, I'm sure (those devil-handers)